### DISCUSSION TITLE
[Projects Beta] How to access archived draft issues?

### DISCUSSION THREAD

**Discussion by hossameldeen** [community/community#8988] [Projects and Issues] [answered]
I wrote a draft issue. I didn't assign it to any repo. I then archived it. Question: Where can I access this issue?

**Context:** Projects Beta seems like a nice way to have a personal todo list. And it'd nice to be able to add tasks/items/draft issues that aren't linked to any specific repositories -- kind-of like Cards in old Projects. However, in old Projects, we could access archived cards, but in Projects Beta, I can't find how to access archived draft issues.

**Comment by iamjonny** [comment, marked as answer]
Good question!

> On the top-right side of the project board, click  Menu.
> Click , then click View archive.
> 

![image](https://user-images.githubusercontent.com/28756257/180564521-6683bd22-9af4-4c28-8436-fd9e8055821a.png)

https://docs.github.com/en/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards/archiving-cards-on-a-project-board#restoring-cards-on-a-project-board-from-the-sidebar

💡 url: `https://github.com/orgs/{org}/projects/{project-id}/archive`
